definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -5642,13 +5614,6 @@
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solConfirmPersonalService",
```

Comment:
Just need to keep this original object in "AuthorisationCaseEvent-prod.json".

Ideally, **if** Jeremy confirms every role should have "R" access. it would be to have the following changes:
inside prod file:
- original authorisations
inside nonprod file:
- caseworker-divorce-solicitor role with R acess
- [PETSOLICITOR] role with "CRU" access
- (No need to include [RESPSOLICITOR] as it would inherit from the roles for 'caseworker-divorce-solicitor'

Reply:
Update: a3120d151a18c4e9120540997088ba6b3402e370

Reply:
Just to clarify, 
```
  {
    "LiveFrom": "02/02/2021",
    "CaseTypeID": "DIVORCE",
    "CaseEventID": "solConfirmPersonalService",
    "UserRole": "caseworker-divorce-solicitor",
    "CRUD": "CRU"
  },
```
Is in the -prod file